optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -823,6 +836,7 @@ def __init__(
             optuna_callbacks=optuna_callbacks,
             verbosity=verbosity,
             show_progress_bar=show_progress_bar,
+            optuna_seed=optuna_seed,
```

Comment:
How about swapping the order of `optuna_seed` and `model_dir` for consistency?

Reply:
Done.